src/components/LoginForm/LoginForm.module.scss

```diff
@@ -0,0 +1,69 @@
+.loginSection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.loginForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 3rem;
+
+  @media (min-width: $screen-md-min) {
+    padding: 0 1rem;
+  }
+}
+
+.signInTitle {
+  text-align: center;
+  margin-bottom: 2rem;
+  font-size: 1.5rem;
+  font-weight: $extrabold;
+  color: $secondary-color;
+
+  @media (min-width: $screen-sm-min) {
+    font-size: 2rem;
+  }
+}
+
+.signInButton {
+  width: 250px;
+  height: 2.75rem;
+  background-color: transparent;
+  border: 1.5px solid $secondary-color;
+  border-radius: 24px;
+  color: $secondary-color;
+  font-size: 1rem;
+  font-weight: $semibold;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $primary-color;
+  }
+
+  @media (min-width: $screen-sm-min) {
+    width: 400px;
+  }
+}
```

Comment:
This, or even more. Is identical to the RegisterForm! We certainly need to abstract it 🙈